<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>771</width>
      <height>411</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd_Data"/>
    <addaction name="actionDelete"/>
    <addaction name="actionFind"/>
    <addaction name="actionSort"/>
    <addaction name="actionReload_data"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCalculator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSort"/>
   <addaction name="actionAdd_Data"/>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="usingMainWindow.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="statusTip">
    <string>Ini untuk new file</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="usingMainWindow.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="statusTip">
    <string>Ini adalah menu open file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionCalculator">
   <property name="text">
    <string>Calculator...</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find...</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="icon">
    <iconset resource="usingMainWindow.qrc">
     <normaloff>:/icons/YouTubeMusic.ico</normaloff>:/icons/YouTubeMusic.ico</iconset>
   </property>
   <property name="text">
    <string>Sort..</string>
   </property>
  </action>
  <action name="actionAdd_Data">
   <property name="icon">
    <iconset resource="usingMainWindow.qrc">
     <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Add Data..</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="usingMainWindow.qrc">
     <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionReload_data">
   <property name="text">
    <string>Reload data...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="usingMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
